geoportal/geoportailv3_geoportal/static-ngeo/js/search/searchDirective.js

```diff
@@ -74,8 +74,10 @@ const exports = function(appSearchTemplateUrl) {
               function(scope) {
                 $(element).find('input').val('').trigger('input');
                 var ctrl = /** @type {app.search.SearchController} */ (scope['ctrl']);
-                ctrl.featureOverlay.clear();
-                ctrl.lastSelectedSuggestion = null;
+                if (ctrl) {
```

Comment:
I had this issue, cases where the scop does not contain the correct controller.
Does this fix the cause or just the symptom ?